tests/rptest/redpanda_cloud_tests/config_profile_verify_test.py

```diff
@@ -82,69 +77,60 @@ def _check_aws_nodes(self):
                 self._clusterId),
             '--query="Reservations[0].Instances[*].InstanceType"'
         ]
-        res = subprocess.check_output(cmd)
-        resd = json.loads(res)
+        output = subprocess.check_output(cmd)
```

Comment:
As a help to future souls - should we just mass dump certain outputs.  When an assertion fails, next step would be: "so what YAML did we get".   Saves a few hours of "add prints / rerun", etc.

Reply:
Good call. Might as well just log it as debug-level output.